_corporate-pages/information-security.markdown

```diff
@@ -0,0 +1,92 @@
+
+### Passwords
+
+All user passwords are stored in our database using Bcrypt, a complex one-way password hashing algorithm that provides protection over common attacks such as rainbow tables and brute-force search attacks.
+
+### Encryption in transit
+
+All communication between you, users using your join form, and any integrations with your internal CRM systems traverse the Internet via encrypted HTTPS traffic using TLS v1.3 were supported. We support older clients with TLS v1.2.
+
+This encryption during communication ensures information cannot be read or manipulated by unauthorised third parties.
+
+### Backups and data retention
+
+We ensure that all data is regularly backed up. A daily backup is taken each day and retained for 7 days. Weekly backups are retained for 4 weeks.
+
+By default we redact all user-submitted personal data in incomplete or unsubmitted applications after 14 days. All user-submitted personal data in submitted applications is redacted 60 days after you have retrieved them for processing. If you feel that these retention periods do not suit you, you can choose to change them at any time.
+
+### Access to data
+
+Join Together staff are granted access to systems and data based on their role in the company or on an as-needed basis. SSO and 2FA are used to ensure access is as secure as possible.
+
+Access to user data by Join Together staff is only used to assist with support and to resolve customer issues.
+
+When working on a support issue we do our best to respect your privacy and the privacy of your members as much as possible and only access the minimum data needed to resolve your issue.
+
+### Communications Security
+
+All Join Together’s transactional emails are sent using Postmark and are encrypted using TLS, ensuring messages are encrypted in transit to remote mail servers and ISPs who support TLS.
+
+Our mail servers are configured to use [DKIM](https://postmarkapp.com/guides/dkim) and [SPF](https://postmarkapp.com/guides/spf) allowing us to control our domain’s reputation, reducing the risk of email spoofing and ensuring a high deliverability rate.
+
+## How we keep our code secure
+
+### Automatic static code analysis
+
+We  employ a continuous integration process that is automatically run against all changes made to the codebase before it can be released. This includes automated static code analysis configured to identify potential vulnerabilities in the code. We also run automated checks to alert us to vulnerabilities in any of the dependent libraries within the application.
+
+### Quality Assurance (QA)
+
+New features and code changes go through a series of stages before they are released to the live system to maintain quality and safety:
+
+- high value code changes are reviewed by a second member of staff to identify potential bugs and issues before it can be released
```

Comment:
```suggestion
- a second member of staff reviews high value code changes to identify bugs and potential issues before releases are approved
```

Switch to active voice, move potential to issues (no such thing as a potential bug!)